prosemirror-jdita/src/document.ts

```diff
@@ -41,6 +53,7 @@ export const NODES: Record<string, (value: JDita, parent: JDita) => any> = {
       });
     }
     const result = { type: value.nodeName, attrs, content: content.map(child => travel(child, value)) };
+    // TODO: Why are we setting attrs here again. It's already set above?
```

Comment:
We should resolve this...

Reply:
Fixing the code seems to be out of scope for this documentation task/PR. I would suggest to do this in a new PR.

Reply:
This question has been moved to a list on the Petal google document, and the `TODO` has been removed from the code.